osf_tests/test_analytics.py

```diff
@@ -92,33 +95,35 @@ def page_counter_for_individual_version(project, file_node3):
 class TestPageCounter:
 
     def test_download_update_counter(self, project, file_node):
-        mock_session = Session()
-        mock_session.data = {}
+        session = SessionStore()
+        session['auth_user_id'] = 'yeji'
```

Comment:
We changed `PageCounter.update_counter` so that it does not save session when the session is anonymous. That why we need `auth_user_id` to be a non-null value. Otherwise, the session does not get saved between the invocations of `PageCounter.update_counter` and would faile the test.